src/FFTW.jl

```diff
@@ -27,14 +27,38 @@ function __init__()
     # libfftw3{,f} refs at runtime, since we may have relocated and
     # changed the path to the library since the last time we precompiled.
     @static if fftw_provider == "fftw"
-        libfftw3[] = FFTW_jll.libfftw3_path
-        libfftw3f[] = FFTW_jll.libfftw3f_path
+        libfftw3_path[] = FFTW_jll.libfftw3_path
+        libfftw3f_path[] = FFTW_jll.libfftw3f_path
         fftw_init_threads()
     end
     @static if fftw_provider == "mkl"
-        libfftw3[] = MKL_jll.libmkl_rt_path
-        libfftw3f[] = MKL_jll.libmkl_rt_path
+        libfftw3_path[] = MKL_jll.libmkl_rt_path
+        libfftw3f_path[] = MKL_jll.libmkl_rt_path
     end
+    return nothing
+end
+
+if VERSION >= v"1.12.0-DEV.1421"
+    # Currently `OncePerProcess` is broken with `Nothing` as the value type which is why
+    # `Int` is used instead (https://github.com/JuliaLang/julia/pull/58155).
```

Comment:
https://github.com/JuliaLang/julia/pull/58155 is closed so this should presumably be fixed.